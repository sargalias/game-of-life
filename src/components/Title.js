import React from 'react';

const Title = (props) => (
  <h1 id="title">{props.title}</h1>
);

export default Title;
