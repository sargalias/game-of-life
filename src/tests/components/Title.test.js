import React from 'react';
import { shallow } from 'enzyme';
import Title from '../../components/Title';

test('should render Title correctly', () => {
  const wrapper = shallow(<Title title="Test title" />);
  expect(wrapper).toMatchSnapshot();
});

test('title prop should appear in rendered component', () => {
  const value = "test title";
  const wrapper = shallow(<Title title={value} />);
  expect(wrapper.find('#title').text()).toBe(value);
});
