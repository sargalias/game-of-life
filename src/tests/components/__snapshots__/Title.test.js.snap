// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Title correctly 1`] = `
<h1
  id="title"
>
  Test title
</h1>
`;
